src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java

```diff
@@ -69,11 +69,28 @@
     public static final String SESSION_SYNC_OBJ = "org.jenkinsci.plugins.ssegateway.sse.session.sync";
     private static final Logger LOGGER = Logger.getLogger(EventDispatcher.class.getName());
 
+    private volatile boolean isRetryLoopActive = false;
+
+    // set lifetime for retry events - default 5min - 300 sec - 300000 msec
+    @edu.umd.cs.findbugs.annotations.SuppressWarnings("MS_SHOULD_BE_FINAL")
+    public static /* not final */ long RETRY_QUEUE_EVENT_LIFETIME = Integer.getInteger(EventDispatcher.class.getName() + ".RETRY_QUEUE_EVENT_LIFETIME", 5*60) * 1000;
```

Comment:
would be better as final. Why not having a separate class for test so you can `@BeforeClass` to set the value?

Reply:
is there any other way to make this value configurable in jenkins?
i have 'stolen' this code from another plugin to be able to change the values via jenkins script console. and, in my understanding, this is not possible if the values are declared as final.

Reply:
nope it's a good way but `static final` would be better